internal/legacy/dockerregistry/schema/manifest_test.go

```diff
@@ -59,11 +59,11 @@ func TestParseThinManifestsFromDirPostsubmit(t *testing.T) {
 			}
 		}
 
-		expectedDigestCount := 12344
+		expectedDigestCount := 14603
 		if onlyProwDiff {
-			expectedDigestCount = 1
+			expectedDigestCount = 14603
```

Comment:
This is because the commit that you picked doesn't promote any image. The way `onlyProwDiff` option works is that it's going to promote/reconcile only images that are promoted in the given commit. Since the given commit doesn't promote any image, it's going to reconcile all images.

I recommend picking some commit that promotes some image. I tried with `e2fd478e1b498d0e87b2ac7e78a1dd53e686551c` and it worked for me.